src/mui/field/ReferenceManyField.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import LinearProgress from 'material-ui/LinearProgress';
+import { crudGetManyReference as crudGetManyReferenceAction } from '../../actions/dataActions';
+import { getReferences, relatedTo } from '../../reducer/references/oneToMany';
+
+/**
+ * Render related records in a list of a single field.
+ *
+ * The child field will be repeated as many times as there are related records.
+ *
+ * @example Display all the books by the current author
+ * <ReferenceManyField reference="books" target="author_id">
+ *     <ChipField source="title" />
+ * </ReferenceManyField>
+ */
+export class ReferenceManyField extends Component {
+    componentDidMount() {
+        this.props.crudGetManyReference(this.props.reference, this.props.target, this.props.record.id, relatedTo(this.props.reference, this.props.record.id, this.props.resource, this.props.target));
```

Comment:
I know you like one liners but this is annoying to review on github.

```js
const relatedField = relatedTo(this.props.reference, this.props.record.id, this.props.resource, this.props.target);
this.props.crudGetManyReference(this.props.reference, this.props.target, this.props.record.id, relatedField);
``` 

same for the others in `componentWillReceiveProps` and `ReferenceManyListField` ?